sync/cli/awsdatabricks.py

```diff
@@ -63,9 +63,7 @@ def run_job(job_id: str, plan: str, compute: str, project: dict = None):
 @click.option("--project", callback=validate_project)
 def create_prediction(run_id: str, plan: str, compute: str, project: str = None):
     """Create a prediction for a job run"""
-    prediction_response = awsdatabricks.create_prediction_for_run(
-        run_id, plan, compute, project["id"]
```

Comment:
little bug here trying to index a string

Reply:
Ah - the type hint on the argument was `str`, so I was going off of that. Sounds like that type hint should be a `dict` instead, then?

Reply:
I see - I missed that `validate_project` callback. Makes more sense now! I was like "how is a user reasonably going to be able to pass a `dict` here?"

Reply:
Yeah, looks like I got the type descriptor wrong